codemeta.json

```diff
@@ -527,7 +527,7 @@
   ],
   "isPartOf": "https://ropensci.org",
   "keywords": ["metadata", "codemeta", "ropensci", "citation", "credit", "linked-data"],
-  "releaseNotes": "https://github.com/ropensci/codemetar/blob/master/NEWS.md",
+  "releaseNotes": "https://github.com/Bisaloo/codemetar/blob/master/NEWS.md",
```

Comment:
Can you remove this edit? (I know it's not your fault :joy:)

Reply:
I can but it will create it again, won't it?

Reply:
Is there a way maybe to skip GitHub actions (similar to `[ci skip]` in the commit message for travis)?

Reply:
aaaaaaaaah right. np I'll merge and it'll get updated again.

I suppose one could add the skip based on the commit message, yes, can you please open an issue suggesting that?